_pages/acknowledgments.md

```diff
@@ -3,6 +3,22 @@
 
 # Acknowledgments
 
-This work has been supported by the [International Neuroinformatics Coordinating Facility](https://www.incf.org/){:target="_blank"}, the [Neuroimaging Data Sharing Task Force](https://web.archive.org/web/20170813183704/http://wiki.incf.org/mediawiki/index.php/Neuroimaging_Task_Force){:target="_blank"}, the Laura and John Arnold Foundation, NIMH Grant R24MH114705, and NSF grant OAC-1760950.
+This work has been done by the community of [individual contributors](https://bids-specification.readthedocs.io/en/stable/appendices/contributors.html). Their work was
+supported by the
+
+Funding Source | Target
```

Comment:
```suggestion
Funding source | Description of the funded contributions 
```

Reply:
I think it sounds better with "Support" since it might be not necessarily in direct money for work (e.g. as with INCF) but with all the logistics etc...